recipes/app2.rb

```diff
@@ -98,4 +102,40 @@
   environment 'RAILS_ENV' => 'production'
   working_directory '/home/replicant/redmine'
   pid_file '/home/replicant/redmine/pids/unicorn.pid'
+  action :stop
```

Comment:
Once you apply the fix I suggested below, you should update this to the following. You might also add a ``# TODO:`` comment here to remove this after the migration has been completed.
```suggestion
  action [:stop, :disable]
```